### PR TITLE
Updated the base-microservice-example to version 2.3.0-alpha

### PR DESCRIPTION
**Pull request by EthanIDR**

### CODE REVIEW

File: src/main/java/conversion/BuildVuServlet.java [hunk 1]
```diff
@@ -139,13 +139,13 @@ protected void convert(Individual individual, Map<String, String[]> params,
     private void setErrorCode(final Individual individual, final int errorCode) {
         switch (errorCode) {
             case 1:
-                individual.setErrorCode(String.valueOf(1050)); // Libreoffice killed after 1 minute
+                individual.doError(1050); // Libreoffice killed after 1 minute
                 break;
             case 2:
-                individual.setErrorCode(String.valueOf(1070)); // Internal error
+                individual.doError(1050); // Internal error
```
Comment (daniel-warren) [inline src/main/java/conversion/BuildVuServlet.java]: @EthanIDR - It seems you have used the same error code for each case. I have added the suggestion to correct this.

```suggestion
                individual.doError(1070); // Internal error
```

File: src/main/java/conversion/BuildVuServlet.java [hunk 2]
```diff
@@ -139,13 +139,13 @@ protected void convert(Individual individual, Map<String, String[]> params,
...
                 break;
             default:
-                individual.setErrorCode(String.valueOf(1100)); // Internal error
+                individual.doError(1050); // Internal error
```
Comment (daniel-warren) [inline src/main/java/conversion/BuildVuServlet.java]:
```suggestion
                individual.doError(1100); // Internal error
```

**Comment by EthanIDR** [comment]
@daniel-warren - I have committed your changes

File: pom.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
     <groupId>com.idrsolutions</groupId>
     <artifactId>buildvu-microservice-example</artifactId>
     <packaging>war</packaging>
-    <version>2.2.0-alpha</version>
+    <version>3.2.0-alpha</version>
```
Comment (daniel-warren) [inline pom.xml]: Could this be changed back to 2.3.0 before merging?

```suggestion
    <version>2.3.0-alpha</version>
```

**Pull request merged**